Local Storage vs Session Storage

```diff
@@ -0,0 +1,39 @@
+The read-only sessionStorage property accesses a session Storage object for the current origin. sessionStorage is similar to localStorage; the difference is that while data in localStorage doesn't expire, data in sessionStorage is cleared when the page session ends.
+
```

Comment:
Could you mention the difference point wise, so that it would be easier to read and understand